src/archiefvernietigingscomponent/js/components/review-list/zaak-detail-modal.js

```diff
@@ -26,9 +26,18 @@ const ZaakDetailModal = ({ modalIsOpen, setIsOpen, zaak, index, comment, setComm
                 <header className="zaak-detail__header">
                     <h1 className="title modal__title">Beoordeel de zaak</h1>
                 </header>
+                {zaak.zac_link !== null &&
                     <div className="zaak-detail__link">
-                        Link naar de zaak in de zaakafhandelcomponent (ZAC): <a href={zaak.zac_link}>{zaak.identificatie}</a>
+                        Zaak:&nbsp;
+                        <a
+                            title="Open de geconfigureerde externe applicatie voor het bekijken van de zaak."
+                            target="_blank"
+                            href={zaak.zac_link}>
```

Comment:
missing quotes i think?
```suggestion
                            href="{zaak.zac_link}">
```

Reply:
I don't think it should have the quotes. It could be ```{`${zaak.zac_link}`}``` but the way it is works!